node/pkg/fetcher/fetcher.go

```diff
@@ -1 +1,142 @@
 package fetcher
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/bus"
+	"bisonai.com/orakl/node/pkg/db"
+	"bisonai.com/orakl/node/pkg/utils"
+)
+
+const FETCHER_FREQUENCY = 2 * time.Second
+
+func NewFetcher(bus *bus.MessageBus) *Fetcher {
```

Comment:
Packages usually include functions related to a single component, therefore it is not always necessary to include the name of the package to function names, because it will be used in the call itself (e.g. `fetcher.New()`).